ebusd-2.1.x/en/vaillant/08.bai.csv

```diff
@@ -25,5 +25,6 @@
 [HW=9602]!load,bai.0010015600.inc,,,
 [HW=8701]!load,bai.0020066007.inc,,,
 [HW=7401]!load,bai.308523.inc,,,
+[HW=4702]!load,bai.protherm25kov.inc,,,
```

Comment:
use the product number instead of the name please (you can get it by using `ebusctl f -d -v -c scan.08 id`)